docs/search-filters.md

```diff
@@ -3,28 +3,33 @@
 The search experience can be improved by adding search filters to narrow down the results. The
 filters rely on listing's indexed data.
 
+* [Filter types](#filter-types)
+* [Adding a new search filter](#adding-a-new-search-filter)
+  * [Common changes](#common-changes)
+  * [Desktop filters](#desktop-filters)
+  * [Desktop filters panel](#desktop-filters-panel)
+  * [Mobile filters](#mobile-filters)
+* [Creating your own filter types](#creating-your-own-filter-types)
+
 ## Filter types
 
-The Flex template for web has three different filter types: _price filter_, _select single_ and
-_select multiple_. The _price filter_ is for the specific case of filtering listings with a price
-range.
+The Flex template for web has different filter types: _price filter_, _date range filter_, _select
+single_ and _select multiple_. The _price filter_ is for the specific case of filtering listings
+with a price range.
```

Comment:
Few notes about the paragraph, the types are listed so we could list: price, date range, select single and select multiple.

Also, the specificity of price filter is mentioned, so maybe the specificity of the date range filter could also be mentioned.